setup.rst

```diff
@@ -125,7 +125,23 @@ On **Fedora** and other ``DNF`` based systems::
    $ sudo dnf builddep python3
 
 On **Debian**, **Ubuntu**, and other ``apt`` based systems, try to get the
-dependencies for the Python version that you're working on e.g.::
+dependencies for the Python using the ``apt``.
```

Comment:
I'm not a native speaker, but the second part of the new sentence reads awkward:

```
[...] try to get the dependencies for the Python using the ``apt``.
```
How about something like "[...] you can get the dependencies for the Python version that you're working on by using the \`\`apt\`\` command"?